src/attributes.jl

```diff
@@ -337,6 +337,18 @@ An optimizer attribute for the string identifying the solver/optimizer.
 """
 struct SolverName <: AbstractOptimizerAttribute end
 
+"""
+    LogLevel(value::Int)
+
+An optimizer attribute for level of verbosity of the optimizer. `0` means
+no output and the meaning is solver specific for positive values.
```

Comment:
Before we just had `silent`. It was unambiguous. Is that better than a weird solver-specific parameter?

Reply:
Good point, I am not sure what's best. The idea of `LogLevel` is that most solver have this functionality, just with a different name. I usually check the MOI tests of the solver to see which name is the log-level.

Reply:
`Silent` addresses the question of "how do I turn the solver output off?", but not "how do I turn solver output on?". `LogLevel` covers both.

Reply:
`Silent(false)`. I'm fine with loglevel though.

Reply:
What's the definition of `Silent(false)`? Would it turn on output or just not silence it?